src/frontend/screens/profile/EditProfile.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import {
+	Box,
+	useColorModeValue,
+	FormControl,
+	FormLabel,
+	Input,
+	InputGroup,
+	InputLeftAddon,
+	Textarea,
+	Button,
+	Flex,
+} from "@chakra-ui/react";
+import { updateUser } from "../../features";
+import { useDispatch, useSelector } from "react-redux";
+
+function EditProfile({ close, user }) {
+	const [profile, setProfile] = useState(user);
+	const dispatch = useDispatch();
+	const { token } = useSelector((state) => state.auth);
+	const handleChange = (e) => {
+		setProfile({
+			...profile,
+			[e.target.name]: e.target.value,
```

Comment:
you can use like 
const {name , value } = e.target
[name] : value

Reply:
Added this changes